libs/blocks/global-navigation/global-navigation.js

```diff
@@ -627,11 +624,13 @@ class Gnav {
 
     // Exposing UNAV config for consumers
     CONFIG.universalNav.universalNavConfig = getConfiguration();
-    window.UniversalNav(CONFIG.universalNav.universalNavConfig);
-
-    isDesktop.addEventListener('change', () => {
-      window.UniversalNav.reload(CONFIG.universalNav.universalNavConfig);
-    });
+    window.UniversalNav(CONFIG.universalNav.universalNavConfig)
+      .then(() => {
```

Comment:
Super minor, but I wonder if this won't be clearer with an `await` instead of a `.then`. That way we we get less nesting.

Reply:
sure. I've updated this to await